mantle/cmd/kola/testiso.go

```diff
@@ -103,14 +103,14 @@ var (
 	}
 	tests_s390x = []string{
 		"iso-live-login.s390fw",
+		"iso-live-login.4k.s390fw",
 		"iso-offline-install.s390fw",
-		// https://github.com/coreos/fedora-coreos-tracker/issues/1434
-		// "iso-offline-install.mpath.s390fw",
-		// https://github.com/coreos/fedora-coreos-tracker/issues/1261
-		// "iso-offline-install.4k.s390fw",
+		"iso-offline-install.4k.s390fw",
+		"iso-offline-install.mpath.s390fw",
 		"pxe-online-install.s390fw",
+		"pxe-online-install.4k.s390fw",
 		"pxe-offline-install.s390fw",
-		"miniso-install.s390fw",
+		"pxe-offline-install.4k.s390fw",
```

Comment:
I don't think we should enable so many 4k tests. The goal here is we get a strategic covering of the possible scenarios without trying to cover them all. i.e. let's add maybe 2 or 3 versus 5. Or maybe we can convert some other tests that are running into 4k ones?